extensions/drt/coding/src/test/resources/valid_offering_property_pairs.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<drt:DeleteResultTemplate version="2.0.0" service="SOS" xmlns:drt="http://www.opengis.net/drt/1.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.opengis.net/drt/1.0 file://tsclient/RemminaDisk/home/eike/Code/git/common-xml/52n-xml-drt-v10/src/main/xsd/drt/1.0/DeleteResultTemplate.xsd">
+	<drt:offering>offering1</drt:offering>
```

Comment:
I would suggest to define a subelement that contains a offering/observedProperty pair.